Add new featurizer based on ConveRT model (#4800)
RasaHQ/rasa: CHANGELOG.rst

  tracker store
- Add command line argument ``rasa x --config CONFIG``, to specify path to the policy and
  NLU pipeline configuration of your bot (default: ``config.yml``)
- Added a new NLU featurizer - ``ConveRTFeaturizer`` based on `ConveRT <https://github.com/PolyAI-LDN/polyai-models>`_ model released by PolyAI.
- Added a new preconfigured pipeline - ``pretrained_embeddings_convert``.

Changed
-------
- Do not retrain the entire Core model if only the ``templates`` section of the domain is changed.
- Upgraded ``jsonschema`` version

> - Renamed message attribute related constants in NLU to remove ``MESSAGE`` prefix.

Reviewer: ```suggestion
- Removed ``MESSAGE`` prefix from the names of message attribute related constants in NLU
```

Reviewer: also please add it to migration guide: if you have custom components ...
